java/code/src/com/redhat/rhn/domain/formula/FormulaFactory.java

```diff
@@ -298,13 +303,18 @@ public static void saveServerFormulaData(Map<String, Object> formData, String mi
      * @return the layout
      */
     public static Optional<Map<String, Object>> getFormulaLayoutByName(String name) {
-        File layoutFileOfficial = new File(METADATA_DIR_OFFICIAL + name + "/form.yml");
-        File layoutFileCustom = new File(METADATA_DIR_CUSTOM + name + "/form.yml");
+        File layoutFileStandalone = new File(METADATA_DIR_STANDALONE_SALT + name + File.separator +  LAYOUT_FILE);
+        File layoutFileManager = new File(METADATA_DIR_MANAGER + name +  File.separator +  LAYOUT_FILE);
+        File layoutFileCustom = new File(METADATA_DIR_CUSTOM + name +  File.separator +  LAYOUT_FILE);
 
         try {
-            if (layoutFileOfficial.exists()) {
+            if (layoutFileStandalone.exists()) {
                 return Optional.of((Map<String, Object>) YAML.load(
-                        new FileInputStream(layoutFileOfficial)));
+                        new FileInputStream(layoutFileStandalone)));
+            }
+            else if (layoutFileManager.exists()) {
+                return Optional.of((Map<String, Object>) YAML.load(
+                        new FileInputStream(layoutFileManager)));
```

Comment:
These things fit on a single line meanwhile, but maybe you want to keep the styling consistent with the existing `else` clause without changing it.